container/deploy_openqa_containers.sh

```diff
@@ -0,0 +1,228 @@
+    if docker logs "$container" 2>&1 | grep "$message" >/dev/null; then
+      echo "OK - Container $container stand-up correctly - found in logs $message"
+      break
+    fi    
+  done
+
+  if [[ $count -eq countMax ]]; then
+    echo "ERROR - Sorry the container $container didn't stand-up correctly"
+    exit 2
+  fi
+}
+
+function show_usage {
+  echo "Usage: $0 [options] <action>"
+  echo "Actions:"
+  echo " - prepare, prepare the environment. If uses --force the current directories will be removed and recreated"
+  echo " - db, create the database container. If uses --force the current db container will be removed and recreated"
+  echo " - webui, create the webui container. If uses --force the current configuration and webui container will be removed and ecreated"
+  echo " - worker, create the worker container If uses --force the current configuration and worker container will be removed and ecreated"
+  echo "Options:"
+  echo "-f|--force, destroys and recreate the previous data or container associated to the action"
+  echo "--key, api key. Required deploying the worker"
+  echo "--secret, api secret. Required deploying the worker"
+  echo "--prefix [network_name], set a prefix network and container creation (the default is openqa)"
+  echo "--port, changes the port for the UI (default 80)"
+  echo "--webui-image, to select what image will be used for the webui container (the default  is registry.opensuse.org/devel/openqa/containers15.2/openqa_webui:latest)"
+  echo "--worker-image, to select what image will be used for the worker container (the default is registry.opensuse.org/devel/openqa/containers15.2/openqa_worker:latest)"
+  echo "Notes:"
+  echo "  For initial deployments execute in order the actions: prepare, db, webui, worker"
+}
+
+function prepare {
+  [[ $FORCE -eq 1 ]] && rm -rf data 2>/dev/null
+  mkdir -p data/factory/{iso,hdd,other,tmp} data/{testresults,tests,conf} data/certs/{ssl.crt,ssl.key}
+  chmod a+w data/testresults
+
+  [[ $FORCE -eq 1 ]] && docker network inspect "$NETWORK" >/dev/null 2>&1 && docker network rm "$NETWORK" >/dev/null
+  docker network create "$NETWORK" >/dev/null || exit 1
+  
+  echo "Next step (db), creation of the data base"
+}
+
+function db {
+  CONT_NAME="${PREFIX}_db"
+
+  [[ $FORCE -eq 1 ]] && set +e && docker rm -f "$CONT_NAME" >/dev/null 2>&1; set -e
+  docker run -d --network "$NETWORK" -e POSTGRES_PASSWORD=openqa -e POSTGRES_USER=openqa -e POSTGRES_DB=openqa --net-alias=db --name "$CONT_NAME" postgres >/dev/null
+
+  wait_for_container_ready "$CONT_NAME"
+  wait_for_container_appropiate_logs "$CONT_NAME" "database system is ready to accept connections"
+
+  echo "Next step (webui), stand-up the webui"
+}
+
+function webui {
+  [[ ! -d data ]] && echo "Please execute first prepare command" && exit 1
+
+  CONT_NAME="${PREFIX}_webui"
+  
+  pushd .
+  cd data/conf || exit 1
+  wget https://raw.githubusercontent.com/os-autoinst/openQA/master/container/webui/conf/openqa.ini >/dev/null
```

Comment:
I'd avoid repeating the same path again and again:
```suggestion
  local openqa_url_prefix=https://raw.githubusercontent.com/os-autoinst/openQA/master
  wget "$openqa_url_prefix/container/webui/conf/openqa.ini" >/dev/null
```